internal/bot/bot.go

```diff
@@ -133,17 +136,20 @@ func (b *Bot) HandleEvent(event interface{}) {
 		}
 		shasString := strings.TrimSpace(shas.String())
 
-		log.Debug().Str("SHA", shasString).Msg("CheckRun received")
-
 		if id := cr.GetApp().GetID(); id != int64(b.appID) {
+			log.Debug().Str("SHA", shasString).Msg("CheckRun received")
 			log.Debug().Str("SHA", shasString).Msgf("Event App ID of %d does not match Bot's App ID of %d", id, b.appID)
 			return
 		}
+
 		if action := event.GetAction(); !lib.Contains(checkRunRelevantActions, action) {
+			log.Debug().Str("SHA", shasString).Msg("CheckRun received")
 			log.Debug().Str("SHA", shasString).Msgf("Unhandled action received: %s. Discarding...", action)
 			return
 		}
 
+		log.Info().Str("SHA", shasString).Msg("CheckRun received")
```

Comment:
should this be log.Debug or log.Info?

Reply:
I made it `Info` since if it gets past those two if statements we know we should care about the event 👍